packages/engine/tests/units/message/remove_agent/src/init-js.json

```diff
@@ -10,5 +10,9 @@
   {
     "agent_name": "2",
     "agent_id": "00000000-0000-0000-0000-000000000002"
+  },
+  {
+    "agent_name": "3",
+    "agent_id": "00000000-0000-0000-0000-000000000003"
   }
```

Comment:
Added the third engine to be removed to test a `"remove_agent"` message to `"Hash"` alongside a message to `"hash"` and `"HASH"`.